### PR TITLE
Fix credentials for GDS

### PR DESCRIPTION
**Pull request by dmohs**
Note: This breaks the DS integration tests, but I spent a while trying to fix them with no luck so I'm going to let them break and revisit later as needed.

### CODE REVIEW

**Comment by dmohs** [comment]
@danqrodney PTAL

File: api/build.gradle [hunk 1]
```diff
@@ -181,17 +185,18 @@ dependencies {
...
+  compile 'com.google.code.gson:gson:+'
+  compile 'com.google.oauth-client:google-oauth-client-jetty:1.23.0'
   compile 'com.squareup.okhttp:okhttp:+'
   compile 'com.squareup.okhttp:logging-interceptor:+'
-  compile 'com.google.code.gson:gson:+'
   compile 'joda-time:joda-time:+'
   compile 'javax.inject:javax.inject:1'
   compile 'io.swagger:swagger-annotations:1.5.16'
-  compile 'com.google.cloud:google-cloud-bigquery:+'
+  compile 'org.json:json:20160810'
```
Comment (danqrodney) [inline api/build.gradle]: Why is this needed?

Comment (dmohs) [reply api/build.gradle]: Compare the result of `./project.rb gradle dependencies --configuration compile` with and without this line. Without it, you get `org.json:json:20160810 -> 20140107` which seems wrong. "This seems better" is as strong as I feel, so I'm happy to remove it.

Comment (danqrodney) [reply api/build.gradle]: I don't know when it's appropriate to override dependencies with a later version like this. In some cases you could be breaking things due to some incompatibility; in most cases (and probably this case) it should be harmless (and may be beneficial, assuming later versions have more bug fixes.)

So there's some risk by doing things like this we inadvertently break things in future, but the risk should be pretty low, and hopefully tests will catch any issues.

Anyway, I have no strong preference here, provided everything keeps working. :)

Comment (dmohs) [reply api/build.gradle]: Okay, I took it out. I don't like listing dependencies we don't explicitly use.
